index.js

```diff
@@ -93,6 +116,7 @@ export default class MapboxAccessibility {
     this.map.on('movestart', this._movestart);
     this.map.on('moveend', this._render);
     this.map.on('render', this._render);
+    this.map.on('movestart', this._geolocate);
```

Comment:
It might be better to watch https://www.mapbox.com/mapbox-gl-js/api/#geolocatecontrol.event:geolocate

Reply:
Hi @andrewharvey I don't believe you can get a reference to that object, hence the hacks. So basically `addControl` doesn't appear to store a reference to the control in `mapbox-gl-js`. The `geolocate` event fires on the `GeolocateControl` not the map. Unless I'm missing something?

Reply:
Oh good point, the GeolocateControl object you create is known by the user, but not the Map object.

Reply:
Yeah I'd like to change this in mapbox-gl-js at some point, but that's for another time